lib/plug/static.ex

```diff
@@ -58,7 +64,15 @@ defmodule Plug.Static do
     at    = Keyword.fetch!(opts, :at)
     from  = Keyword.fetch!(opts, :from)
     gzip  = Keyword.get(opts, :gzip, false)
-    cache = Keyword.get(opts, :cache, true)
+    cache = Keyword.get(opts, :cache_control_for_query_strings)
+
+    # TODO Remove in 1.0
+    if Keyword.has_key?(opts, :cache) do
```

Comment:
Nah, we don't need deprecations. Just remove it and we will add an entry to the changelog.